tests/unit/schema/test.firefox-schemas-import.js

```diff
@@ -1150,7 +1150,6 @@ describe('firefox schema import', () => {
         .withArgs('manifest.json')
         .returns(true);
       sinon.stub(fs, 'createReadStream').withArgs(inputPath).returns(zipfile);
-      sinon.stub(fs, 'unlinkSync').withArgs(inputPath).returns(undefined);
```

Comment:
@willdurand This is the only change required to fix the tests. Would you mind giving this a look to make sure it's ok?

Reply:
This seems weird. I don't see why it is still there. It comes from this revision: https://github.com/mozilla/addons-linter/commit/02d2a70771812590b7241ce4d812cfb4790e464b, but the actual code is clearly not the same nowadays.

@rpl thoughts?

Reply:
@willdurand yeah Chris did migrate the schema import from tar to zip files some times ago as part of #3036, and then in #3095 we did tweak this test file to ensure that we remove the zip file used to test the schema import. In #3095 we introduced the `prepareTmpDir` that is now triggering that "cannot delete non-empty directory" (when the cleanup function defined by it is going to be called in the afterEach hook).

I guess that something changed internally in tmp, tmp-promise or in rimraf which is now making the stubbed unlinkSync to actually prevent the files inside that directory from being removed as expected, and that ends up triggering that "cannot delete non-empty directory" error.

I don't see why we should stubbing unlinkSync, and so in my opinion the fix applied by @wagnerand is ok.